cocos/game/game.ts

```diff
@@ -307,6 +307,12 @@ export class Game extends EventTarget {
      */
     public static readonly EVENT_RESUME = 'game_on_resume';
 
+    /**
+     * @en Events triggered when the game pauses to resume running. <br>
```

Comment:
Whether there is any problem in the remarks

Reply:
Yes, I remember that it was modified before, it should be an error when merging